larky/src/main/java/com/verygood/security/larky/modules/crypto/CryptoHashModule.java

```diff
@@ -23,6 +19,7 @@
 import org.bouncycastle.crypto.digests.SHAKEDigest;
 import org.bouncycastle.crypto.digests.KeccakDigest;
 import org.bouncycastle.crypto.util.DigestFactory;
+import org.bouncycastle.jcajce.provider.digest.Keccak;
```

Comment:
we don't really need this: https://github.com/bcgit/bc-java/blob/f9cfd83a947f065dec3f4d997bb9a6373f9e9e25/prov/src/main/java/org/bouncycastle/jcajce/provider/digest/Keccak.java#L17-L52.

We can re-write.